feat(menu/compareied): Add new menu option to compare a IED with one from template project

dlabordus opened the pull request:
closes #836

JakobVogelsang commented:
Hi @dlabordus I have a first look on the functionality and have some little UI remarks and one bigger functional one that I think we need to discuss:

**UI**
- first dialogs title is the same one as the second and is misleading if you ask me. It should say something like `Select IED for comparison`, `IEDs auswählen`
- title of each of the columns is plural `IEDs ...`, `IEDs im Projekt`, `IEDs aus Vorlage`
- the german translation for the button is better `Starte Vergleich`
- second dialog noninteractive header shows only one IED in my case. Say I load two IEDs with different IED names it will say `Child elements from IED(AA1E1Q01BCU)` but below that also children of the other one are organized. I do not know if this is on purpose, as they differ by the icon. With files with a lot of changes this is misleading, though, I would argue.
- I was thinking that the identity string in `()` on the second page could be in secondary span as well. They tend to be very long the dialog has an unnecessary with sometimes

dlabordus commented:
Hi @JakobVogelsang,

Reaction on your comments in bold:

- first dialogs title is the same one as the second and is misleading if you ask me. It should say something like Select IED for comparison, IEDs auswählen
**All 3 dialog have their own title now, that's more clear indeed. But now a extra German title is needed :-)**
- title of each of the columns is plural IEDs ..., IEDs im Projekt, IEDs aus Vorlage
**Changed it**
- the german translation for the button is better Starte Vergleich
**I think that was already there, but please check**
- Second dialog noninteractive header shows only one IED in my case. Say I load two IEDs with different IED names it will say Child elements from IED(AA1E1Q01BCU) but below that also children of the other one are organized. I do not know if this is on purpose, as they differ by the icon. With files with a lot of changes this is misleading, though, I would argue.
**I accidentally switch the two element. I also changed the names of the variables, but that stays a little but difficult. Also I made the intermediate titles bold, this way it more clear where the changes below to. Please check if it's now more clear.**
- I was thinking that the identity string in () on the second page could be in secondary span as well. They tend to be very long the dialog has an unnecessary with sometimes
**I changed it, but the identity is not bold, but these looks kind of oke.**

JakobVogelsang reviewed (approved):
LGTM. Nice to read thank you
